src/2024/5e-SRD-Feats.json

```diff
@@ -0,0 +1,1228 @@
+[
+  {
+    "index": "ability-score-improvement",
+    "name": "Ability Score Improvement",
+    "prerequisites": [],
+    "desc": [
+      "General Feat",
```

Comment:
What purpose does this serve? The SRD doesn't contain this text in the Feat description.

We probably need to add a `feat_type` field to differentiate Origin, General, Fighting Style and Epic Boon Feats..

Reply:
I'm aware that you mention this in your PR description, but we should probably be designing this data nicely all in one go, rather than breaking it into lots of PRs.